### ISSUE TITLE
Advanced Pluralization issue

### ISSUE THREAD

**Issue by anderssonola** [rmariuzzo/Laravel-JS-Localization#37]
The following is defined in php and is working fine in my blade templates:
`'apple_count' => '{0} :i apples |[1,1] :i apple |[2,Inf] :i apples'`

```
@choice('apple_count', 0, ['i' => 0]) prints "0 apples"
@choice('apple_count', 1, ['i' => 1]) prints "1 apple"
@choice('apple_count', 2, ['i' => 2]) prints "2 apples"
```

Doing the same with JS-Localization, doesn't work as expected, it seams the JS version only supports the 'string1|string2|string3' syntax?
```
Lang.choice('apple_count', 0, {'i':0}); //prints "0 apples"
Lang.choice('apple_count', 1, {'i':1}); //prints "1 apples"
Lang.choice('apple_count', 2, {'i':2}); //prints "2 apple"
```

**Comment by rmariuzzo** [comment]
Will check today as soon as I can. In q couple of hours.

**Comment by kae0zz** [comment]
Hey there,
ist there any update on this issue? :)

Looks like it's a problem of the standard rules of the choice function:

```javascript
// Standard rules
if (count > 1) {
    return messageParts[1];
} else {
    return messageParts[0];
}
```
Instead of just checking if count > 1, it should consider the given parameter - please see Laravel Pluralization and the possibility to define counts as {0}, {1} and so on. Thanks! :)

**Comment by a-komarev** [comment]
You could try test it with [all locales pluralization PR](https://github.com/rmariuzzo/Lang.js/pull/27) if you wish.

**Comment by rmariuzzo** [comment]
The fix contributed by @a-komarev was merged into `develop` branch. This will be shipped in v1.1.0 soon.
